dash_labs/plugins/pages.py

```diff
@@ -470,13 +470,12 @@ def _parse_query_string(search):
 
     parsed_qs = {}
     for (k, v) in parse_qs(search).items():
-        first = v[0]  # ignore multiple values
+        v = v[0] if len(v) == 1 else v
         try:
-            first = json.loads(first)
+            v = json.loads(v)
```

Comment:
What's the use case for json loading the value here? And in particular, do we need to `json.loads` each item, if we have a list at this point?

Reply:
@chriddyp can you answer this question about the use-case for json.loads?

Reply:
JSON parsing - to bring number strings into numbers and allow human readable booleans (`true`, `false`)

Reply:
OK, good - that's convenient.  Will just need to document it because "false", "true", "null" will convert, but "False", "True" and "None" remain as strings.

Reply:
@alexcjohnson should we also use this with the path variables for consistency? 

If so, should the default path when using a `path_template` be `"/dasboard/null"` rather than `"/dashboard/none"` ?

Reply:
>  "False", "True" and "None" remain as strings.

Oh true, good point. That feels a bit confusing, I could imagine that we convert those to the Python types as well

Reply:
After giving this a little more thought, I think it might be better to keep data as strings.  For example, not all string numbers should be converted to int or float.  What if the number was part of a 3 digit string id?  `"?id=010"`  would be converted as a string `{"id": "010"}` but `"?id=100"`  would be converted as an int `{"id": 100}` . 


Plus there are other keywords that would convert that  people might not expect --  such as "NaN" and "Infinity"

Reply:
@chriddyp Is this good to go now?  I removed the `json.loads.`